test/util/framework/hcp_helper.go

```diff
@@ -294,8 +328,15 @@ func CreateOrUpdateExternalAuthAndWait(
 
 	switch m := any(operationResult).(type) {
 	case hcpsdk20240610preview.ExternalAuthsClientCreateOrUpdateResponse:
-		// TODO someone may want this return value.  We'll have to work it out then.
-		//fmt.Printf("#### got back: %v\n", spew.Sdump(m))
+		// Make sure the operation result content matches the current external auth model.
```

Comment:
minor, but still required: please add the details of your slack conversation with me as a comment on each of these to explain what we're testing